webrender/src/tiling.rs

```diff
@@ -595,10 +600,7 @@ fn add_to_batch(
                                 BlendMode::SubpixelConstantTextColor(text_cpu.font.color.into())
                             }
                         }
-                        GlyphFormat::Alpha |
-                        GlyphFormat::TransformedAlpha |
-                        GlyphFormat::Bitmap |
-                        GlyphFormat::ColorBitmap => BlendMode::PremultipliedAlpha,
+                        _ => BlendMode::PremultipliedAlpha,
```

Comment:
any particular reason for this change? In general, providing all the enum values is preferred for issuing an error whenever the enum changes.